Library/Homebrew/cmd/audit.rb

```diff
@@ -225,9 +246,35 @@ def audit_urls
       problem "\"ftpmirror.gnu.org\" is preferred for GNU software (url is #{u})."
     end
 
-    # the rest of the checks apply to mirrors as well
+    # the rest of the checks apply to mirrors as well.
     urls.concat(@specs.map(&:mirrors).flatten)
 
+    # Check a variety of SSL/TLS links that don't consistently auto-redirect.
+    urls.each do |p|
+      # Skip the main url link, as it can't be made SSL/TLS yet.
+      next if p =~ %r[/ftpmirror\.gnu\.org]
+
+      # No auto-direct as of Jan 2015.
```

Comment:
Likewise, all of these should read ` No auto-redirect `. Will fix post-feedback.